Completion/Unix/Type/_tar_archive

```diff
@@ -20,8 +20,10 @@ if [[ "$1" = *[urtx]* ]]; then
     _files "$expl[@]" -g '*.(tar|TAR).bz2(-.)'
   elif [[ "$1" = *J* ]]; then
     _files "$expl[@]" -g '*.(tar|TAR).(lzma|xz)(-.)'
+  elif [[ "$1" = *zstd* ]]; then
+    _files "$expl[@]" -g '*.(tar|TAR).(zst|zstd)(-.)'
   elif [[ "$_cmd_variant[$service]" == (gnu|libarchive) ]]; then
-    _files "$expl[@]" -g '*.((tar|TAR)(.gz|.GZ|.Z|.bz2|.lzma|.xz|)|(tbz|tgz|txz))(-.)'
```

Comment:
Have you verified with both libarchive and GNU tar? I only have the libarchive variant available to me and it appears to have a --zstd option with no short form.

Reply:
Yeah, I checked: https://github.com/libarchive/libarchive/blob/2f3033ca23f8c21160506c3c7ac8a0df0d3fde42/cpio/bsdcpio.1#L189

Reply:
GNU tar supports --zstd too, with no short form either.